src/sentry/tasks/post_process.py

```diff
@@ -1118,6 +1123,12 @@ def process_code_mappings(job: PostProcessJob) -> None:
             org_slug = org.slug
             next_time = timezone.now() + timedelta(hours=1)
 
+            # php automatic code mappings currently in LA
+            if event.data["platform"].startswith("php") and not features.has(
```

Comment:
`platform` is not guaranteed in the event, thus, this could raise an error.

I do not know if the pipeline is designed to prevent barfing from such an error being thrown.

Reply:
Good call, the newest commit makes the default behavior to skip the platform check if `platform` doesn't exist

Reply:
This looks like it'd already be failing if so since make the same access a few lines up: https://github.com/getsentry/sentry/pull/66151/files#diff-b8b58ee6be5cf471f56bfbcf8d91a64bdfca7ae9800245997d6eb5acfff13aceR1108

Reply:
@MichaelSun48  @mrduncan that check few lines up doesn't quite solve it. It can be `None`, which will blow up:

```python
>>> data = {"platform": None}
>>> bool("platform" in data)
True
>>> data["platform"].startswith("php")
AttributeError: 'NoneType' object has no attribute 'startswith'
```

Reply:
Good catch! Resolved in [c8c1639](https://github.com/getsentry/sentry/pull/66151/commits/c8c163910ce7c8b9f4bb3935dbbb5d4faaeb65d8)